.github/workflows/product-research.yml

```
@@ -96,6 +98,20 @@ jobs:
     - name: Build project
       run: pnpm run build
 
+    - name: Write ASINs to Job Summary
+      if: github.event.inputs.asins != ''
+      env:
+        INPUT_ASINS: ${{ github.event.inputs.asins }}
+      run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:31: Double quote to prevent globbing and word splitting

(shellcheck)